### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by angoyd**
Added attribute endpoint and div project endpoints and some fixes.

### CODE REVIEW

File: Visma.net/Models/Project.cs [hunk 1]
```diff
@@ -31,12 +37,20 @@ public Project()
         public string description { get { return Get<string>(); } set { Set(value); } }
         public DateTime startDate { get { return Get<DateTime>(); } set { Set(value); } }
         public DateTime endDate { get { return Get<DateTime>(); } set { Set(value); } }
-        [JsonProperty]
-        public ProjectManager projectManager { get { return Get<ProjectManager>(); } private set { Set(value); } }
+
+        public ProjectManager projectManager {
```
Comment (omelhus) [inline Visma.net/Models/Project.cs:41]: Why are you setting employeeNumber but getting projectManger (sic)? Is this an error?

File: Visma.net/Models/AttributeDetails.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using ONIT.VismaNetApi.Lib;
+using ONIT.VismaNetApi.Models.CustomDto;
+using ONIT.VismaNetApi.Models.Enums;
+
+namespace ONIT.VismaNetApi.Models
+{
+    public class AttributeDetails : DtoProviderBase
```
Comment (omelhus) [inline Visma.net/Models/AttributeDetails.cs:12]: Should this have an override for PrepareForUpdate? Or is it not possible to update attributes?

**Review by omelhus — CHANGES_REQUESTED** [review]

File: Visma.net/Models/Project.cs (hunk 1 above)
Comment (angoyd) [reply Visma.net/Models/Project.cs:41]: When getting project from visma, you gets meta info of the project manager
image
but when posting project project is just an emp id.
image

Could maybe be solved an other way?

File: Visma.net/Models/AttributeDetails.cs (hunk 2 above)
Comment (angoyd) [reply Visma.net/Models/AttributeDetails.cs:12]: Missed will fix that.

Comment (angoyd) [reply Visma.net/Models/AttributeDetails.cs:12]: Looked at it now and I actually did it in the Attribute class. (models\Attribute.cs)

Comment (omelhus) [reply Visma.net/Models/AttributeDetails.cs:12]: Good!

File: Visma.net/Models/Project.cs (hunk 1 above)
Comment (omelhus) [reply Visma.net/Models/Project.cs:41]: Maybe ProjectManager should be a IBecomeDto with `ToDto() => new DtoValue(employeeNumber)`?

Comment (angoyd) [reply Visma.net/Models/Project.cs:41]: I Changed the ProjectManager class now.

**Review by omelhus — APPROVED** [review]

**Pull request merged**